src/Standards/Squiz/Sniffs/PHP/InnerFunctionsSniff.php

```diff
@@ -45,39 +46,26 @@ public function process(File $phpcsFile, $stackPtr)
             return;
         }
 
-        $conditions = $tokens[$stackPtr]['conditions'];
+        $reversedConditions = array_reverse($tokens[$stackPtr]['conditions'], true);
```

Comment:
Performance hats-up: for nested arrays, in most cases performance of array functions is better when you first slice off the bit you want to work with.

So:
``` php
$conditions = $tokens[$stackPtr]['conditions'];
$reversedConditions = array_reverse($conditions, true);
```
... should give a little better performance.

Reply:
Interesting, never heard this one before. Updated just now.

Reply:
Have a look at this PR where some similar replacements in a much used sniff, lowered the memory usage of PHPCS for a complete run over a codebase by > 6%: https://github.com/squizlabs/PHP_CodeSniffer/pull/2273